.github/workflows/ci.yml

```diff
@@ -57,6 +57,27 @@ jobs:
       - name: Run tests
         run: ./vendor/bin/phpunit --testsuite Unit
 
+  build80:
```

Comment:
I think it was because of the different Symfony versions in `sf_version` that were not compatible with php8 due to some reason, but I will check again this evening (european time).

Reply:
I removed the exclusive php 8 section and added it to the existing one. Works like a charm. I cannot recall what I thought at that time, but the less the better : ) Thanks!